meeting-notes/2018/2018-08-20--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 August 20, 2018
+
+- **Lead:** Dominic Della Valle
+- **Notetaker:** 
+  - @bigs
+- **Attendees:**
+  - @mgoelzer
+  - @keks
+  - @djdv
+  - @stebalian
+  - @kevina
+  - @hsanjuan
+  - @Magik6k
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+ 
+@djdv
+ - Done:
+   - Refined the IPFS on fuse implementation
+     - Added caching around things like pins, keys, and metadata
+     - Reduced lock contention
+     - Reworked how indexes are generated and referenced
+    - Added fs stat data
+      - free space remaining on daemon's datastore device
+      - available file descriptors/handles
+    - Partial symlink support
+      - they work with *nix tools and Go's os pkg but not win32
+ - Blocked:
+   - Previously I wanted to work on write support, but was blocked on an issue related to file permissions
+     - This was resolved while refactoring
+   - Currently I'm dealing with an issue related to file ownership
+     - I am currently in contact with the cgofuse developer about it
+     - Currently working around this by setting everything to 0777, later node owners will be the same as the process owner/invoker
+ - Next:
+   - Add create, write, and delete methods for MFS
+   - Experiment with mutable methods in non-mutable areas
+     - rmdir/del -> unpin; for files directly under the ipfs root
+     - write -> add -w
+ 
+@diasdavid (Unsure if will have good connectivity for this one)
+ - Done:
+   - Done more interviews (almost there, only 3 that still need to be scheduled)
+   - Request DX Team to get go-ipfs, go-libp2p and go-ipld Packages table https://github.com/ipfs/testing/issues/128
+ - Blocked:
+ - Next: 
+   - Finish the interviews/chats
+   - Compile all the notes from the interviews/chats into something easy to consume by everyone in the Go Core Dev Teams
+   
+@kevina
+  - Done:
+    - In Progress: Begin new "cid" sub-command https://github.com/ipfs/go-ipfs/pull/5385
```

Comment:
@kevina Is this for CIDv0 to CIDv1 migration? Please create an issue on http://github.com/ipfs/specs so that the other IPFS implementations and Client libraries can keep up.

//cc @alanshaw @hugomrdias @achingbrain

Reply:
This was something that I consider go specific and something I want to put into a spec.  One of the things it does is to provide a way to convert CIDs to base32 CIDv1.  I considered making it a separate utility but that will be a separate binary that might not be installed.  If it was part of `go-ipfs` it is nearly guaranteed to be available when if its a separate utility